movie/src/components/Features/DetailPopup/index.jsx

```diff
@@ -1,20 +1,20 @@
-import { Grid, Typography } from "@mui/material";
-import ArrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos";
+import KeyboardArrowRightIcon from "@mui/icons-material/KeyboardArrowRight";
 import "./DetailPopup.css";
 const DetailPopup = ({ handleClose }) => {
   return (
-    <Grid className="detailPopup" onClick={handleClose} fullWidth>
-      <Typography>Want to rate or add this item to a list?</Typography>
-      <Grid className="card">
-        <button className="labelBtn">Login</button>
-        <ArrowForwardIosIcon />
-      </Grid>
-      <Typography>Not a member?</Typography>
-      <Grid className="card">
-        <button className="labelBtn">Sign up and join the community</button>
-        <ArrowForwardIosIcon />
-      </Grid>
-    </Grid>
+    <div className="detailPopup" onClick={handleClose}>
```

Comment:
I dont recommed using `onClick` on `<div />`

Reply:
I have change it to button.